Tests/Resources/config.php

```diff
@@ -15,7 +15,7 @@
         'base_url' => 'http://localhost',
         'sahi' => array(),
         'zombie' => array(),
-        'selenium' => array(),
+//        'selenium' => array(),
```

Comment:
It worked before (or otherwise why it's written like this), why it doesn't work for you?